docs/xshareclasses.md

```diff
@@ -124,16 +124,16 @@ When you specify `-Xshareclasses` without any parameters and without specifying
 
 : If you set this suboption to 0000, the default directory permissions are used. If you set this suboption to 1000, the machine default directory permissions are used, but the sticky bit is enabled.
 
-: If the cache directory is the platform default directory, `/tmp/javasharedresources`, this suboption is ignored and the cache directory permissions are set to 0777.
+: If the cache directory is the platform default directory, `<HomeDir>/.cache/javasharedresources`, this suboption is ignored and the cache directory permissions are set to 0777.
```

Comment:
Not sure if we want to explicitly mention `<HomeDir>` means user's home directory here. It might be obvious to most readers, but it does not hurt doing so.